BLocal/BLocal.csproj

```diff
@@ -62,8 +62,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Shared.Management.Objects.for.SQL.Server.2008.R2.11.0.3000.0\lib\Microsoft.SqlServer.SqlEnum.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.6.0.5\lib\net35\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@KrzychuR should it be: `Version=9.0.0.0` -> `Version=9.0.1` ?

Reply:
@rusln I have no idea why VS/NuGet added 9.0.0.0 with reference to ..\packages\Newtonsoft.Json.9.0.1\lib\net35\Newtonsoft.Json.dll :) I've used NuGet Package Manager for solution and package manager reporting version 9.0.0.1 :(